### ISSUE TITLE
WSODs/error 500 when redis caching enabled

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#5336]
I had redis caching enabled via UNIX socket like this...

````
$a->config['system']['cache_driver'] = 'redis';
$a->config['system']['redis_host'] = '/var/run/redis/redis.sock';
$a->config['system']['redis_port'] = 0;
````

It was working very well until one of the latest changes to the cache drivers. (or lock fixing?)
I don't know exactly. But it started approx. yesterday.

### Steps to reproduce the problem
Enable redis caching and go to some profile pages or try to post something. My profile page was a WSOD. Another user had the same problem. Also another node showed the same behavior.

After disabling redis support in .htconfig.php everything is working again.

### Friendica version you encountered the problem
2018.08-dev

### Friendica source (git, zip)
git

### PHP version
7.0

### SQL version
10.0 MariaDB

**Comment by MrPetovan** [comment]
Duplicate of #5333

**Comment by MrPetovan** [comment]
Closed via #5339
